extensions/resteasy-common/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-extensions-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-resteasy-common-spi-parent</artifactId>
+    <name>Quarkus - JMS SPI - Parent</name>
```

Comment:
```suggestion
    <name>Quarkus - RESTEasy Common SPI - Parent</name>
```

Reply:
I would also add a description indicating that this module is intended to be shared between resteasy classic and quarkus rest (aka Resateasy reactive)